Configuration/Geometry/python/GeometryRecoDB_cff.py

```diff
@@ -5,6 +5,7 @@
 
 #Tracker
 from RecoTracker.GeometryESProducer.TrackerRecoGeometryESProducer_cfi import *
+from Geometry.TrackerGeometryBuilder.TrackerAdditionalParametersPerDet_cfi import *
```

Comment:
Again, in these core config files for Runs 2 & 3, please add the comment I mentioned for `GeometryExtended_cff.py`.

Reply:
I've added the comments in the 4 files where they were requested (+ changed the PR title/description).